specification/managedservices/resource-manager/readme.go.md

```diff
@@ -0,0 +1,26 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: managedservices
+  clear-output-folder: true
+```
+
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: package-2018-06-01-preview
```

Comment:
should be `package-2018-06-preview` per https://github.com/Azure/azure-rest-api-specs/pull/5328/files#diff-20f9c7f8021cc64c275ac54a421744dcR33

Reply:
Made the suggested change.